skins/oasis/css/core/SpeechBubble.scss

```diff
@@ -50,9 +50,20 @@
 			overflow: hidden;
 			width: 553px;
 		}
-        .toc ul {
-            list-style-type: none;
-        }
+
+		.toc ul {
+			list-style-type: none;
+		}
+
+		figure {
+			height: auto;
+			max-width: 100%;
+		}
+
+		.image > img {
```

Comment:
Is this to prevent images from getting bigger than max width regardless of the available height? In other words: scale image towards max available height but stop scaling once they reach max available width?